src/components/admin/order/TitledOrderStatusList.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { memo } from 'react';
+import styled from '@emotion/styled';
+import { Order, OrderStatus } from '@@types/index';
+import { colFlex, rowFlex } from '@styles/flexStyles';
+import AppLabel from '@components/common/label/AppLabel';
+import { Color } from '@resources/colors';
+import NotPaidOrderCard from './NotPaidOrderCard';
+import PaidOrderCard from './PaidOrderCard';
+import ServedOrderCard from './ServedOrderCard';
+
+const OrderCardListContainer = styled.div`
+  ${colFlex({ align: 'start' })}
+  gap: 10px;
+  width: 100%;
+  overflow-x: auto;
+`;
+
+const CardListContainer = styled.div`
+  ${rowFlex()}
+  gap: 10px;
+  height: 180px;
+`;
+
+interface OrderStatusListProps {
+  orders: Order[];
+  title: string;
+}
+
+function areOrdersEqual(prevOrders: Order[], nextOrders: Order[]) {
+  if (prevOrders.length !== nextOrders.length) return false;
+
+  return prevOrders.every((prevOrder, index) => {
+    const nextOrder = nextOrders[index];
+
+    const hasSameOrderId = prevOrder.id === nextOrder.id;
+
+    const areProductsEqual = prevOrder.orderProducts.every((prevProduct, productIndex) => {
+      const nextProduct = nextOrder.orderProducts[productIndex];
+      const hasSameServedStatus = prevProduct.isServed === nextProduct.isServed;
+      const hasSameServedCount = prevProduct.servedCount === nextProduct.servedCount;
+      return hasSameServedStatus && hasSameServedCount;
+    });
+
+    return hasSameOrderId && areProductsEqual;
```

Comment:
```suggestion
	return areOrderEqual(prevOrder, nextOrder);
```

이미 있는 util 함수를 사용해도 괜찮을 것 같은데 따로 작성하신 이유가 있나요?

Reply:
넵 그래서 내부에서 order 단위로 비교하는 부분에 대해서 대체가 가능하지 않은가? 라고 여쭤본겁니다~

Reply:
아하 대체 가능하네요! 💯